app/app.go

```diff
@@ -724,16 +724,3 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 
 	return paramsKeeper
 }
-
-// extractRegisters isolates the encoding module registers from the module
-// manager, and appends any solo registers.
-func extractRegisters(m sdkmodule.BasicManager) []encoding.ModuleRegister {
-	// TODO: might be able to use some standard generics in go 1.18
-	s := make([]encoding.ModuleRegister, len(m))
-	i := 0
-	for _, v := range m {
-		s[i] = v
-		i++
-	}
-	return s
-}
```

Comment:
This implementation is unnecessarily complex. Refactored and moved to modules.go because it is only used once in that file.